includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-courses.php

```diff
@@ -57,21 +57,51 @@ public function get_items( array $filters ): array {
 			'suppress_filters' => 0,
 		);
 
+		if ( ! empty( $filters['orderby'] ) && ! empty( $filters['order'] ) ) {
+			$course_args['orderby'] = $filters['orderby'];
+			$course_args['order']   = $filters['order'];
+		}
```

Comment:
This looks like a duplicate. Aren't the sorting args already added in the array a [few lines back](https://github.com/Automattic/sensei/blob/ae101e8f1fc2d3fb8657526e8d5b790f019b35a4/includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-courses.php#L55-L56)?

Reply:
Oops, missed that moment 😬

Reply:
Totally I've removed it. [here](https://github.com/Automattic/sensei/pull/4970/commits/a411bd74121681db267ee86ce3838b70265feec6)